src/components/swap/SwapBox.vue

```diff
@@ -49,6 +49,7 @@
                 placeholder="0.00"
                 v-model="swapTo.value"
                 @change="handleSwapOutput"
+                @focus="clearSwapTo"
```

Comment:
We had agreed to remove the clean in both the from and the to, right?

Reply:
done